doc/spec/package-lock.md

```diff
@@ -28,15 +28,6 @@ The version of the package this is a package-lock for. This must match what's in
 An integer version, starting at `1` with the version number of this document
 whose semantics were used when generating this `package-lock.json`.
 
-### packageIntegrity *(new)*
```

Comment:
This was a hash of the `package.json` which would result in churn and merge conflicts on the `package-lock.json`. It was never implemented and the consensus of the discussion is that it was a bad idea.